nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -123,6 +123,10 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The `ssh` client tool now disables the `~C` escape sequence by default. This can be re-enabled by setting `EnableEscapeCommandline yes`
 
+- The `ssh` module does not read `/etc/ssh/ssh_known_hosts2` anymore since this location is [deprecated since 2001](https://marc.info/?l=openssh-unix-dev&m=100508718416162&w=2).
+
+- `sshd` does not read `~/.ssh/authorized_keys2` anymore since this location is [deprecated since 2001](https://marc.info/?l=openssh-unix-dev&m=100508718416162&w=2).
```

Comment:
The `sshd` service?

Reply:
This could mention that the behavior can easily be reverted by setting some options… then again, I'm sure the 2 people that actually still need this will manage to figure it out.